src/main/java/org/fundacionjala/sfdc/pageobjects/contacts/SFCNewModifyPage.java

```diff
@@ -1,7 +1,8 @@
 package org.fundacionjala.sfdc.pageobjects.contacts;
 
 
-import org.fundacionjala.sfdc.pageobjects.SalesForceConnection;
+import org.fundacionjala.sfdc.pageobjects.BasePage;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.fundacionjala.sfdc.pageobjects.BasePage'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)